lib/livebook/session/data.ex

```diff
@@ -1109,7 +1109,7 @@ defmodule Livebook.Session.Data do
       notebook: Notebook.delete_cell(data.notebook, cell.id),
       bin_entries: [
         %{
-          cell: cell,
+          cell: Map.replace(cell, :outputs, []),
```

Comment:
If you expect :outputs to be there:

```suggestion
          cell: Map.replace!(cell, :outputs, []),
```

Btw, should we have a test or it is not that straightforward?

Reply:
It's intentional, it could be a markdown cell :)

Reply:
I will add a test!

Reply:
Updated!